code/nrf-connect/samples/zigbee/Kconfig

```diff
@@ -6,4 +6,12 @@ config PRST_ZB_SLEEP_DURATION_SEC
 
 config PRST_ZB_PARENT_POLL_INTERVAL_SEC
 	int "Interval for when b-parasite polls its parent for data (in seconds)"
-	default 60
\ No newline at end of file
+	default 60
+
+config BUILD_DATE
```

Comment:
As I understand the `BUILD_DATE` property, it is expected a 1-byte string and suggests to have a date stamp. 

One could either provide a fixed default date in code for people who don't care about it or maybe set it to the date of fixed github releases each time a new firmware is released?

In my local workflow I usually override it so that I can distinguish the firmware build on the deployed parasites...

Reply:
That makes sense. I will make sure to add `-DPRST_ZB_BUILD_DATE` to the github action that builds this sample too.

Reply:
Changed the naming of the config flags as discussed.